jcommune-service/src/main/java/org/jtalks/jcommune/service/GroupService.java

```diff
@@ -53,7 +54,7 @@
      */
     List<Group> getByName(String name);
 
-    List<UserDto> getGroupUsers(long id, int count);
+    Page<UserDto> getPagedGroupUsers(long id, String page, int pageSize);
```

Comment:
Page is String because of PageRequest constructor parameter type. For some reason it's String and it's already used in many places of code.